.travis.yml

```diff
@@ -196,10 +202,15 @@ matrix:
 #    - UNSET_S3_SECRETS=1
 
 before_install:
-  - if [ ! -z "${_DL_CRON:-}" ]; then
-      if [ ! ${TRAVIS_EVENT_TYPE} = "cron" ]; then
-        echo "Exiting since should only be ran on a weekly basis by cron!";
-        exit 0;
+  - if [ "${TRAVIS_EVENT_TYPE}" = "cron" ]; then
+      if [ -z "${_DL_CRON:-}" ]; then
+          echo "Skipping regular job on weekly cron";
+          exit 0;
+      fi
+    else
+      if [ ! -z "${_DL_CRON:-}" ]; then
+          echo "Exiting since should only be ran on a weekly basis by cron!";
```

Comment:
Why? Cron should still run all the other tests, eg if we don't push for a while and have a complete pass

Reply:
> On the two days of the week that cron jobs run, I regularly hit into this.

could you please pickup and finish #4640 to use conda instead of bundled annex.  That should speed up our tests runs, helping to mitigate our general problem of lengthy tests run.  Sure thing if others vote (poll?) to make cron run only cron jobs, I would go with majority ;-)

Reply:
> could you please pickup and finish #4640 to use conda instead of bundled annex. That should speed up our tests runs, helping to mitigate our general problem of lengthy tests run.

I doubt that will be a large enough speed up to really matter in this context.  But no, right now I'm focused on figuring out the Travis ssh hang with newer versions of git-annex (and making some progress).

> Sure thing if others vote (poll?) to make cron run only cron jobs, I would go with majority ;-)

No, I would prefer most PR discussions not turn into polls.

This PR isn't urgent.  I'll let it sit another day, and if no one else cares to weigh in, I'll drop the tip commit and merge.

Reply:
> I doubt that will be a large enough speed up to really matter in this context

according to https://github.com/datalad/datalad/issues/4535#issuecomment-629386381 it could be around 40% speed up - almost making it half the run time, so in the same time we could run almost twice as many tests

>  right now I'm focused on figuring out the Travis ssh hang with newer versions of git-annex (and making some progress).

whoohoo -- that is awesome and indeed more pressing!